### ISSUE TITLE
Feature suggestion - Chocolatey and Scoop Windows releases

### ISSUE THREAD

**Issue by andis-sprinkis** [hismailbulut/Neoray#36]
Integrate downloading and putting of Neoray in `%PATH%` for
Scoop - https://scoop.sh/
Chocolatey - https://chocolatey.org/

Although this may also raise general question about Neoray releases on other platforms and Linux distros, Microsoft Windows is where I personally find having a dedicated nvim GUI client like Neoray extra useful, so it would be also useful if I could install it via either of the automated tools mentioned above along with other programs.

**Comment by hismailbulut** [comment]
I agree with that. I am familiar with scoop and I will add Neoray to extras bucket soon, but I don't know how to add to chocolatey. I appreciate any help here. I don't worry about linux package managers because I think most of the linux users just compiles Neoray themselves. But I also want to add Neoray to the homebrew which also I don't know how to add. So this issue has 3 missions.

Scoop -> I can add myself
Chocolatey -> Help wanted
Hombrew -> Help wanted

**Comment by hismailbulut** [comment]
I created a pr for scoop but unfortunately Neoray doesn't meet the requirements for now
We must wait until 100 stars and 50 forks in order to add Neoray to scoop extras
https://github.com/ScoopInstaller/Extras/pull/8966

**Comment by yashamon** [comment]
The fork requirement is weird.

**Comment by hismailbulut** [comment]
It's now added to the scoop extras bucket. I think they removed the 100 stars and 50 forks requirement.
